scripts/build/windows.bat

```diff
@@ -226,6 +233,10 @@ signtool verify /pa /v %package_output%\Etcher.exe
 
 upx -9 %package_output%\*.dll
 
+cd %package_output%^
+ && 7z a -tzip ..\installers\%package_name%.zip *^
```

Comment:
> @jviotti From a technical point of view, you've listed 7-zip as a runtime dependency, and I guess strictly speaking it's only a packaging dependency.

Yeah, the build scripts take care of both initializing the project for development purposes, and dealing with packaging, which is inconvenient. I propose to handle it in the build script refactoring, and get it fixed there for once and for all.

***

I propose to merge this PR to get the ball rolling. With the refactoring @lurch is working on, the script will not complain about packaging-specific dependencies, which means the required dependencies needed to get the project up and running for development purposes will be very few.

In the case of packaging, we're looking forward to move all that to CI servers (code-signing and security policies from the company are stopping us a bit here, but we'll get that sorted), which means very few people will actually do it on their machines anyway.